python-checks/src/test/resources/checks/unusedFunctionParameter/unusedFunctionParameter.py

```diff
@@ -161,3 +161,35 @@ def param_referenced_in_comment_no_issue_3(unused_param_2):
     def param_accessed_through_pandas_no_issue(sample_df, area_of_interest): # OK
         sample_df.query('area == @area_of_interest').population
         print("hello")
+
+import abc
+class Test(abc.ABC):
```

Comment:
Generally, we try to name the test classes/methods with names that represent what is tested. I'd also recommend avoiding calling stuff `Test` whenever possible, since some rules define heuristics to avoid raising issues on test code ([SONARPY-1630](https://sonarsource.atlassian.net/browse/SONARPY-1630) being the latest example) and you wouldn't want to trigger one accidentally (this is not the case here, just a general principle).

I'd suggest names such as:

```
class SomeAbstractClass(abc.ABC):
  def no_issue_on_abstract_class_methods(self, name):
    print("Hello")
```